code/modules/mining/mine_items.dm

```diff
@@ -192,3 +192,61 @@
 	icon_opened = "miningcaropen"
 	icon_closed = "miningcar"
 
+/*********************Mob Capsule*************************/
+
+/obj/item/device/mobcapsule
+	name = "lazarus capsule"
+	desc = "It allows you to store and deploy lazarus-injected creatures easier."
+	icon = 'icons/obj/mobcap.dmi'
+	icon_state = "mobcap0"
+	w_class = 1.0
+	throw_range = 20
+	var/mob/living/simple_animal/captured = null
+	var/colorindex = 0
+
+/obj/item/device/mobcapsule/Destroy()
+  if(captured)
+    qdel(captured)
+    captured = null
+  return ..()
```

Comment:
this isn't properly indented; did you just straight up copy-pasta, to a "T" what I put in hastebin? =p

BYOND uses different tabbing intervals than hastebin ;p